pkg/cmd/pulumi/util.go

```diff
@@ -626,7 +626,12 @@ func getUpdateMetadata(
 
 // addPulumiCLIMetadataToEnvironment enriches updates with metadata to provide context about
 // the system, environment, and options that an update is being performed with.
-func addPulumiCLIMetadataToEnvironment(env map[string]string, flags *pflag.FlagSet, getEnviron func() []string) {
+func addPulumiCLIMetadataToEnvironment(
+	env map[string]string,
+	projectRoot string,
```

Comment:
This function isn't actually using projectRoot?

Reply:
Good catch--this is leftover from an earlier version of these changes. Will remove.